packages/falso/src/lib/pronoun.json

```diff
@@ -0,0 +1,3 @@
+{
+  "data": ["They/Them", "She/Her", "He/Him", "Ze/Hir"]
```

Comment:
Why they are not sepreated?

Reply:
Sorry, webstorm must have done it when I committed. I'm out but I'll sort it tomorrow morning 😊

Reply:
I looks like this is done by a prettier rule that cant be changed. [There's a big discussion about it here.](https://github.com/prettier/prettier-vscode/issues/352)

You can only get around it by being a bit hacky and adding a blank comment to the first line

Reply:
I'm asking about the `/` - `They/Them`

Reply:
Ah sorry I misunderstood what you meant. This is how pronouns are formatted on forms.

So a user might select "She/Her" as their proffered pronouns 😊

Reply:
I'm not familiar with this concept. Do you have an example?

Reply:
So in [this blog post](https://blog.asana.com/2018/05/new-profile-setting-pronouns-asana/) asana are explaining why they've added the pronouns field to their user profile form, and why it's free text.

If you scroll 1/2 way down to the "How to set your pronouns in Asana" section, there's a screen shot 😊

Off the top of my head, Instagram (freetext) and LinkedIn (fixed drop down) also both have similar a field on their user profile form